store/tikv/2pc.go

```diff
@@ -465,7 +470,6 @@ const maxTxnTimeUse = 590000
 
 // execute executes the two-phase commit protocol.
 func (c *twoPhaseCommitter) execute() error {
-	ctx := goctx.Background()
```

Comment:
The contexts used for normal prewriting and cleanup are different and should be separated. It's correct to make this right at the first place even though we use `Background()` for both at now.